dist/pom.xml

```diff
@@ -2,7 +2,7 @@
 
   <modelVersion>4.0.0</modelVersion>
   <name>Zeebe Distribution</name>
-  <artifactId>camunda-cloud-zeebe</artifactId>
+  <artifactId>zeebe-distribution</artifactId>
```

Comment:
Why revert the name of the module?

Reply:
I thought it's not needed, but learned today in the Operate/Tasklist release that it is required. I will revert this change